api/key/key_test.go

```diff
@@ -202,3 +206,61 @@ func TestInfer(t *testing.T) {
 		}
 	}
 }
+
+// Note: The tests below load a real SDK
```

Comment:
strongly prefer a separate file for benchmarks.

Reply:
Done.